### DISCUSSION TITLE
'default' is not exported by rappid/dist/rappid.js

### DISCUSSION THREAD

**Discussion by Iteatechnology** [clientIO/joint#1961] [Q&A] [answered]
We succefully import jointjs+ trial version in our Vue3+Vite+Typescript project, but failed  the formal licenses version.
We can load rappid js in dev environment, but failed to build.
Please check the issue, many thanks!

**Comment by jamesgeorgewilliams** [comment, marked as answer]
You may have to adapt your imports slightly with `vite`.

Try specifying the exact file in your import by using:

`import { dia, shapes } from '@clientio/rappid/rappid.js';` 

instead of 

`import { dia, shapes } from '@clientio/rappid';`

Using imports as above and the following Vite `vue-ts` template should work with both trial and non-trial jointjs+ versions.
https://vitejs.dev/guide/

At the time of writing this template is using the latest version of `vite` which is `4.0.0`.

Both dev and production builds will work with the scripts provided in this template:

```js
{
  "scripts": {
    "dev": "vite", // start dev server, aliases: `vite dev`, `vite serve`
    "build": "vite build", // build for production
    "preview": "vite preview" // locally preview production build
  }
}
```

**Comment by babyandy0111** [comment]
wow, good job. thanks your answer. I solved.
